tests/test.c

```diff
@@ -474,6 +474,7 @@ struct Test tests[] = {
     TEST_CASE_EXPECTED(unlink_error, 1),
     TEST_CASE_EXPECTED(trap_exit_flag, 1),
     TEST_CASE_COND(test_stacktrace, 0, SKIP_STACKTRACES),
+    TEST_CASE_EXPECTED(small_big_ext, 0),
```

Comment:
We don't need _EXPECTED if the test case returns 0.

Reply:
👍🏽